dot_agents/skills/create_handoff/SKILL.md

```
@@ -0,0 +1,96 @@
+for example (between <example_response></example_response> XML tags - do NOT include these tags in your actual response to the user)
+
+<example_response>
+Handoff created and synced! You can resume from this handoff in a new session with the following command:
+
+```bash
+/resume_handoff thoughts/shared/handoffs/ENG-2166/2025-01-08_13-44-55_ENG-2166_create-context-compaction.md
+```
+</example_response>
+
+---
+##.  Additional Notes & Instructions
```

markdownlint-cli2 (0.20.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)